src/components/nav/user.tsx

```diff
@@ -6,14 +6,14 @@ interface IUserInfo {
   balance?: number;
   address?: string;
   eth_address?: string;
-  is_aggregator: boolean;
+  role: string;
 }
 
 const User: React.FC = () => {
   const [info, setInfo] = useState<IUserInfo>({
     username: '--',
     avatar: `${process.env.PUBLIC_URL}/nav/avatar.png`,
-    is_aggregator: false,
+    role: 'aggregator',
```

Comment:
I think it's better to set the init role to `user` to avoid the start icon show when login succeed.

Reply:
done